src/styles/blocks/about.scss

```diff
@@ -0,0 +1,34 @@
+.about {
+  font-family: $main-font-family;
+  color: $main-text-color;
+
+  @include page-grid;
+
+  &__title {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 1 / 3;
+    }
+
+    @include on-desktop {
+      grid-column: span 6;
+    }
+  }
+
+  &__content {
+    grid-column: 1 / -1;
+    font-family: $secondary-font-family;
+    font-weight: 400px;
+    font-size: 16px;
+    line-height: 22.4px;
```

Comment:
avoid using half pixels (they can generally cause problems and sometimes are just not counted, i can only think of them working in box-shadow)